### DISCUSSION TITLE
Alternative for business shared folders with no "Add shortcut to My files" option

### DISCUSSION THREAD

**Discussion by amanjitsk** [abraunegg/onedrive#2892] [Q&A] [answered]
Hi, 

Thanks for the amazing piece of software. For the 2.5.x versions, I see that the business shared folders syncing feature has been changed, and the new docs seems to indicate that the prerequisite to use this feature is to "Add shortcut to My files" for the folders we'd like to sync, and then follow the other instructions.

My problem is that my Onedrive account (its a school account) does not actually have this option/feature, so I can't really add a shortcut to shared business folders in my files. This is apparently a well-known issue affecting many users - see [this](https://answers.microsoft.com/en-us/msoffice/forum/all/no-one-drive-web-option-for-add-shortcut-to-my/ff17bdca-95f9-4d81-aff4-c1e987fa9f6c) for instance. 

Is there an alternative way to sync such business folders (and files - although for files there seems to be another option - Option 2 in the docs that might work) under versions 2.5.x onwards? The older version (and instructions) for 2.4.x works fine for me, but is obviously missing a lot of the new features like relocation/renaming business folders etc.

Best,
Amanjit

**Comment by abraunegg** [comment, marked as answer]
There is no option other than to use the add shortcut method.

Please speak to your school. This is nothing that can be solved here based on the v2.5.x rewrite and how the code now operates.
